tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -57,7 +57,8 @@ spec:
       if [ "$CREATED_CLUSTER" == "" ]; then
         aws eks create-cluster --name $(params.cluster-name) --region $(params.region) --kubernetes-version $(params.kubernetes-version) --role-arn $SERVICE_ROLE_ARN --resources-vpc-config subnetIds=$subnets,securityGroupIds=$sg $ENDPOINT_FLAG
       fi
-      aws eks $ENDPOINT_FLAG --region $(params.region) wait cluster-active --name $(params.cluster-name) 
+      aws eks $ENDPOINT_FLAG --region $(params.region) wait cluster-active --name $(params.cluster-name)
+      sleep 300 
```

Comment:
Line [60](https://github.com/awslabs/kubernetes-iteration-toolkit/pull/440/files#diff-566fabd5277b56d45d422840baf5a353ec32543ea2f755ad6aa45eca934830deR60 ) `wait cluster-active` wouldn't ensure cluster is active already ?